compiler/rustc_trait_selection/src/traits/query/evaluate_obligation.rs

```diff
@@ -92,6 +120,9 @@ impl<'tcx> InferCtxtExt<'tcx> for InferCtxt<'tcx> {
         &self,
         obligation: &PredicateObligation<'tcx>,
     ) -> EvaluationResult {
+        // Run canonical query. If overflow occurs, rerun from scratch but this time
```

Comment:
This comment is a leftover from 2fd5516272e44b08ec01fc48bf6987e9a3409b71, and should have been moved to this function instead of left in `evaluate_obligation`, where this behavior notable does *not* occur.